src/main/java/pack/service/impl/PersonService.java

```diff
@@ -1,59 +1,46 @@
 package pack.service.impl;
 
-import java.util.ArrayList;
 import java.util.List;
-
-import javax.annotation.PostConstruct;
-
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import pack.model.Person;
 import pack.repository.IPersonRepository;
 import pack.service.IPersonService;
+import pack.util.CustomException;
 
 @Service
 public class PersonService implements IPersonService {
 
 	@Autowired
 	private IPersonRepository repository;
-	
-	private List<Person> personList;
-
-	@PostConstruct
-	public void init(){
-		personList=new ArrayList<>();
-	}
 
 	@Override
-	public List<Person> getPersons() {
+	public List<Person> getPersons() throws CustomException {
 		try {
-			for (Person person : repository.findAll()) {
-				personList.add(person);
-			} 
-			return personList;
+			return (List<Person>) repository.findAll();
```

Comment:
Is this cast necessary? The repository shouldn't be using type parameter?

Reply:
yes, 
I get a compilation error without this cast.

"Type mismatch: cannot convert from Iterable<Person> to List<Person>"